src/App.js

```diff
@@ -1,14 +1,20 @@
-import React from "react";
+import React,{Component} from "react";
 import Navigation from "./components/navbar";
 import SimpleFooter from "./components/footer";
+import TfVis from "./components/tf";
 
 
-export default function App() {
-  return (
-    <div>
-      <Navigation />
-      {/* <Home /> */
-      <SimpleFooter /> }
-    </div>
-  );
-}
\ No newline at end of file
+class App extends Component {
+
+  render(){
+    return(
+      <div>
+      <header><Navigation/></header>
+      <TfVis/>
+      <footer><SimpleFooter/></footer>
```

Comment:
remove header/footer tags, it should automatically load as header/footer without extra html tags